bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandler.java

```diff
@@ -149,7 +150,7 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
             configuration.put(configurationParmeter.getKey(), configurationParmeter.getValue());
         }
 
-        if (thingIsInitialized()) {
+        if (ThingHandlerHelper.isHandlerInitialized(this)) {
```

Comment:
As binding developers will need this feature as well, I would prefer to have a  protected`isInitialized()` method on the BaseThingHandler (which then in turn can call the ThingHandlerHelper). This would make it much easier for developers as they would not have to find out about the existence of ThingHandlerHelper at all.